install-panel.sh

```diff
@@ -762,10 +815,21 @@ function goodbye {
   echo "* "
   echo "* Installation is using $WEBSERVER on $OS"
   echo "* Thank you for using this script."
-  echo -e "* ${COLOR_RED}Note${COLOR_NC}: This script does not configure any firewalls for you. 80/443 (HTTP/HTTPS) is required to be open."
+  echo -e "* ${COLOR_RED}Note${COLOR_NC}: If you haven't configured the firewall. 80/443 (HTTP/HTTPS) is required to be open."
   print_brake 62
-
-  exit 0
+  echo ""
+  read -p " Install the Daemon on this machine (y/N): " CONFIRM
+  if [ "$CONFIRM" == "y" ]; then
+    bash <(curl -s https://raw.githubusercontent.com/VilhelmPrytz/pterodactyl-installer/master/install-daemon.sh)
+  elif [ "$CONFIRM" == "Y" ]; then
+    bash <(curl -s https://raw.githubusercontent.com/VilhelmPrytz/pterodactyl-installer/master/install-daemon.sh)
+  elif [ "$CONFIRM" == "n" ]; then
+    exit 1
+  elif [ "$CONFIRM" == "N" ]; then
+    exit 1
+  else
+    exit 1
+  fi
```

Comment:
I like this functionality. Perhaps this can be moved outside of the `goodbye` function?